portfolio/src/main/webapp/script.js

```diff
@@ -38,54 +38,101 @@ function defaultPage() {
   document.getElementById("defaultOpen").click();
 }
  
-/**
- * Fetches messages from the servers and adds them to the DOM.
- */
-async function getComments() {
-    fetch('/data').then(response => response.json()).then((msgs) => {
-   
-    const statsListElement = document.getElementById('comments-container');
-    statsListElement.innerHTML = '';
+// Posts and fetches messages from the server and adds them to the DOM.
+async function sendAndGet() {
+  // Get post parameters.
+  const textEle = document.getElementById('text-input');
+  const nameEle = document.getElementById('sender');
+  const tagEle = document.getElementById('tags');
+
+  // To load the recent post BlogHashMap needs a loadFactor(limit) of 1.
```

Comment:
If I understand this correctly, loadFactor isn't a good variable name. You use loadFactor in the servlet to change numberOfComments, so the variable name should reflect that. Also, load factor has a specific meaning for hash maps/tables, and so you should only use that if you are change the load factor of the hash map.

Load Factor for a hash map isn't the maximum number of occupied spots to empty spots before a hash map increases its size. If your load factor is 0.5, then a hash map will increase its size just after 50% of its spaces are filled.